sklearn/tree/_classes.py

```diff
@@ -537,7 +574,7 @@ def _prune_tree(self):
         check_is_fitted(self)
 
         if self.ccp_alpha < 0.0:
-            raise ValueError("ccp_alpha must be greater than or equal to 0")
+            raise ValueError("must be >= 0.0.")
 
         if self.ccp_alpha == 0.0:
             return
```

Comment:
With your `check_scalar` we will not have this case because we don't include 0.0